pages/products/linear.md

```diff
@@ -468,11 +462,61 @@ ML can also be used as a framework to generate new multigrid methods. Using ML
 
 ML is compatible with Epetra only.
 
-### **MueLu**: multigrid framework
+### **Amesos**: direct sparse linear solver interface
 
-Points-of-contact: Ray Tuminaro (rstumin@sandia.gov), Jonathan Hu (jhu@sandia.gov), and Chris Siefert([csiefer@sandia.gov](mailto:csiefer@sandia.gov))  
-[https://trilinos.github.io/muelu.html](muelu.html)
+Point-of-contact: Siva Rajamanickam (srajama@sandia.gov)  
+[https://trilinos.github.io/amesos.html](amesos.html)
+
+Amesos is a set of C++ interfaces to serial and parallel sparse direct solvers. Amesos contains two nice sparse solvers: KLU and Paraklete. KLU is serial, while Paraklete (distributed with Trilinos 7.0 or higher) is a parallel solver. Amesos also offers an interface to LAPACK, and several other well-know solvers available on the web.
+
+The main idea of Amesos is to give a high-level view of direct solvers, as composed by four main phases:
+
+1) specification of parameters  
+2) initialization of the solver, using matrix sparsity only  
+3) computation of the factors  
+4) solution of the linear system
+
+Amesos insulates the user from all the low-level details typical of direct solvers, like the matrix format, data distribution for the matrix, the solution and the right-hand side, parameter settings, and so on. Amesos is not based on any matrix format; instead, an matrix interface (specified by using the Epetra_RowMatrix class) is adopted. This facilitates the usage of Amesos classes in any projects whose matrix can be wrapped as an Epetra_RowMatrix.
+
+### **Ifpack**: Point preconditioning, incomplete factorizations, and classical domain decomposition
+
+Points-of-contact: Mike Heroux (maherou@sandia.gov) and Siva Rajamanickam (srajama@sandia.gov)  
+[https://trilinos.github.io/ifpack.html](ifpack.html)
+
+Ifpack provides a suite of object-oriented algebraic preconditioners. Ifpack constructors expect an Epetra_RowMatrix object for construction. Ifpack objects interact well with other Trilinos classes. In particular,Ifpack can be used as a preconditioner for AztecOO and smoother in ML.
+
+Ifpack contains one-level domain decomposition preconditioners of overlapping type. Each “subdomain” is defined by the set of rows assigned to a given processors. Several options are available for the local solution, ranging from simple relaxation schemes, to incomplete factorizations, to direct solvers (through the Amesos package).
+
+Ifpack is compatible with Epetra only.
+  
+### **Pliris**: direct dense linear solver
```

Comment:
@srajama1 I am just curious. Can you add Adelus in this page too?

Reply:
@vqd8a I noticed Adelus was missing too. I just did rerrangement of the pages in this. I was going to ask you to add Adelus under direct solvers after we push this :). Since you are here, can you check the changes and approve the PR.

Reply:
Sure, I can add Adelus :-)